### PR TITLE
Ignore today in daily average since the day is not complete

### PR DESCRIPTION
**Pull request by karimkawambwa**
## Description

This change will ignore today in daily average since the day is not complete.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: sensorsafrica/api/v2/views.py [hunk 1]
```diff
@@ -153,7 +153,7 @@ def _retrieve_past_24hrs(city_slugs, filter_value_types):
     def _retrieve_range(from_date, to_date, city_slugs, filter_value_types):
         if not to_date:
             from_date = beginning_of_day(from_date)
-            to_date = end_of_today()
+            to_date = beginning_of_today()
```
Comment (kilemensi) [inline sensorsafrica/api/v2/views.py]: May be just a comment here that what we want is end of y'day

File: sensorsafrica/tests/test_sensordatastats_view.py [hunk 2]
```diff
@@ -49,7 +49,9 @@ def test_getting_air_data_now_all_cities(self, client, sensorsdatastats):
         assert results[2]["city_slug"] == "nairobi"
 
     def test_getting_air_data_now_filter_cities(self, client, sensorsdatastats):
-        response = client.get("/v2/data/air/?city=dar-es-salaam,bagamoyo", format="json")
+        response = client.get(
+            "/v2/data/air/?city=dar-es-salaam,bagamoyo", format="json"
```
Comment (kilemensi) [inline sensorsafrica/tests/test_sensordatastats_view.py]: What is "bagamoyo" doing here :man_facepalming: ... kidding.

File: sensorsafrica/tests/test_sensordatastats_view.py [hunk 3]
```diff
@@ -88,9 +90,17 @@ def test_getting_air_data_from_date(self, client, sensorsdatastats):
 
         data = response.json()
 
-        assert type(data["results"][0]["P1"]) == list
         assert type(data["results"][0]["P2"]) == list
 
+        # Data is in descending order by date
+        last_value = data["results"][0]["P2"][0]
```
Comment (kilemensi) [inline sensorsafrica/tests/test_sensordatastats_view.py]: I think you mean `recent_value` or `most_recent_value` here

File: sensorsafrica/tests/test_sensordatastats_view.py (hunk 2 above)
Comment (karimkawambwa) [reply sensorsafrica/tests/test_sensordatastats_view.py]: 🤔

**Pull request merged**